pkg/scheduler/framework/types.go

```diff
@@ -363,9 +363,14 @@ type QueuedPodInfo struct {
 	*PodInfo
 	// The time pod added to the scheduling queue.
 	Timestamp time.Time
-	// Number of schedule attempts before successfully scheduled.
-	// It's used to record the # attempts metric and calculate the backoff time this Pod is obliged to get before retrying.
+	// Number of all schedule attempts before successfully scheduled.
+	// It's used to record the # attempts metric.
 	Attempts int
+	// Number of the scheduling attemts that this Pod gets unschedulable.
```

Comment:
Seems conflict with the comment above

```
		// This Pod is rejected by some plugins, not coming back due to unexpected errors (e.g., a network issue)
```

Reply:
How does it conflict?
We increment `UnschedulableCount` when `pInfo.UnschedulablePlugins.Union(pInfo.PendingPlugins).Len() != 0`, which is when the pod is coming back because it gets unschedulable, not coming back due to unexpected errors such as networking issues.

Reply:
OK. I was confused by "by some plugins" before, but after rechecking the comments, it is clear to me now.